<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形控制</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>660</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>工作模式</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>波形参数配置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>功放开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>功放关</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>参数读取</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>开始发射</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>调试模式</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>接收器设置</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>采样频率(MSPS)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="amplingFrequency">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20MHz</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>接收增益</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="gainMultiplier">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>128</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>采样长度(k)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上传波形起点(k)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上传波形终点(k)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="waveGetStart">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="waveGetEnd">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="amplingLength">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="maxLength">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>参考增益</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>1/8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3/8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1/2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5/8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3/4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7/8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>发生换能器设置</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射波形</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射次数</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emissionCount">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射换能器通道</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>波形周期数</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射电压(V)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="emissionWavePool">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>高斯波</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>正弦波</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="emissionN">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="emissionInterval">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emissionVoltage">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射频率(kHz)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emissionFrequency">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发射间隔(s)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="waveformCyclesN">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>指令下发</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>参数读取</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startSample">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>690</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始发射</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>211</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>功放设置</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>功率设置</string>
       </property>
      </widget>
      <widget class="QComboBox" name="powercomboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>21</y>
         <width>69</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>低档</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中档</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高档</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>功放开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>功放关</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="channel1" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>931</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="channel2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>931</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileSaveButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>750</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择波形文件存储路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileStream">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>750</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/waveFiles</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="readParameter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>660</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1235</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSYS">
    <property name="title">
     <string>SYS</string>
    </property>
    <addaction name="actionbackground"/>
   </widget>
   <addaction name="menuSYS"/>
  </widget>
  <action name="actionbackground">
   <property name="text">
    <string>background</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
